### ISSUE TITLE
New directive: Preferred-Languages

### ISSUE THREAD

**Issue by joker314** [lirantal/express-security-txt#50]
We have a new directive that might be merged into the live document soon. :tada: 

This directive was originally proposed in securitytxt/security-txt#131, and it will be implemented by securitytxt/security-txt#138 (not merged yet). 

It's syntax is:
```
Preferred-Languages: en, es, ru
```

It shows what languages the security analysts would understand reports best in.

I haven't looked too deeply into it, but I would assume it can only appear once. If so, we could probably use arrays to allow someone to pass in multiple languages? Like with comments, if we'd see a comma inside an array element, we would split that array element into two pieces. Is this behaviour intuitive? 

With validation, we might want to depend on a package which has a list of all the language codes. (Maybe joi already has one built in).

**Comment by lirantal** [comment]
I've been following that too, seems like there's still discussion around it and which standard specs to use exactly for the languages.

Sounds good to me, I think we should add.
I wonder if we want to use a string array or instead preffer an array of objects so we will be future safe in-case some new metadata will be decided upon, for example:

```
const prefLanguages = [
{
  language: 'en',
  comment: 'some comment about it',
  metadata: 'some new metadata key about it',
},
...
]
```

**Comment by joker314** [comment]
I vote string array. Even if later we do need to allow objects, we can still accept both strings and objects and handle the former as meta-data-less internally to keep things working. :)

**Comment by lirantal** [comment]
Alrighty!

**Comment by joker314** [comment]
Closing as resolved by #53 and appears in 2.5.0 release
